examples/access-control-migration/src/services/casbin.enforcers.ts

```diff
@@ -0,0 +1,36 @@
+import * as casbin from 'casbin';
+import path from 'path';
+
+export async function getCasbinEnforcerByName(name: string): Promise<casbin.Enforcer | undefined> {
+  if (name == 'admin') return createAdminEnforcer();
+  if (name == 'owner') return createOwnerEnforcer();
+  if (name == 'team') return createTeamEnforcer();
```

Comment:
As the implementation for `createAdminEnforcer()`, `createOwnerEnforcer()` and `createTeamEnforcer()` is highly the same, maybe you want to pass the configuration file as an parameter to an `createEnforcer()` function?

Reply:
refactored in commit https://github.com/strongloop/loopback-next/pull/4571/commits/4ed9b73865c1bb8143a88bc022fcc89f506ab519